src/Core/src/Platform/Android/JavaObjectExtensions.cs

```diff
@@ -14,15 +15,15 @@ public static bool IsDisposed(this global::Android.Runtime.IJavaObject obj)
 			return obj.Handle == IntPtr.Zero;
 		}
 
-		public static bool IsAlive(this Java.Lang.Object? obj)
+		public static bool IsAlive([NotNullWhen(true)] this Java.Lang.Object? obj)
 		{
 			if (obj == null)
 				return false;
 
 			return !obj.IsDisposed();
 		}
 
-		public static bool IsAlive(this global::Android.Runtime.IJavaObject? obj)
+		public static bool IsAlive([NotNullWhen(true)] this global::Android.Runtime.IJavaObject? obj)
```

Comment:
@PureWeen This looks like an unrelated change to the one of the PR. Is a GOOD change but adding it in this PR could put you in a bad situation. Imaging that you have to revert this commit because it introduced a bug, it will also remove this GOOD but unrelated changed. I would move it to a new smaller PR just with the use of the attribute because:

1. Will allow you to revert
2. Will give better history and a better context for the change.
3. The rest of the team will lear about the attr.